### PR TITLE
StorageSync : Clone existing ApiVersion 2018-04-02 for creating a new version 2018-07-01

### PR DESCRIPTION
**Pull request by ankushbindlish2**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by marstr — CHANGES_REQUESTED** [review]
@ankushbindlish2 ask me to take a look at this offline. 

At this stage, the expected path for the directory `2018-07-01` is:

$REPO_ROOT/specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-07-01/

In addition, I'd update the README here as well to include the new tag. That way the Automation for the various languages above will light-up.

**Comment by marstr** [comment]
I just went ahead and sniped this from @annatisch since I have more context.

**Comment by ankushbindlish2** [comment]
Updated with review comments

File: specification/storagesync/resource-manager/readme.ruby.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
-  - tag: package-2018-07-01
```
Comment (marstr) [inline specification/storagesync/resource-manager/readme.ruby.md]: It is still appropriate to include this tag as part of `multiapi` languages.

**Review by marstr — CHANGES_REQUESTED** [review]
Mind adding a section in the README for `$(go)` while you're at it?

File: specification/storagesync/resource-manager/readme.ruby.md (hunk 1 above)
Comment (ankushbindlish2) [reply specification/storagesync/resource-manager/readme.ruby.md]: Incorporated

File: specification/storagesync/resource-manager/readme.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ These are the global settings for the Storage Sync API.
 
 ``` yaml
 openapi-type: arm
+tag: package-2018-07-01
```
Comment (marstr) [inline specification/storagesync/resource-manager/readme.md]: At this level, only one tag can be specified. It should continue to be the most recent API Version in `stable`

**Review by marstr — CHANGES_REQUESTED** [review]

File: specification/storagesync/resource-manager/readme.md (hunk 2 above)
Comment (ankushbindlish2) [reply specification/storagesync/resource-manager/readme.md]: Should I keep tag: package-2018-07-01 only ?

**Review by marstr — APPROVED** [review]

**Review by marstr — APPROVED** [review]

**Pull request merged**